applications/GeoMechanicsApplication/tests/cpp_tests/test_geomechanics_erosion_piping.cpp

```diff
@@ -0,0 +1,215 @@
+// KRATOS___
```

Comment:
I understand the meaning of erosion_piping in the file name, but would expect the file name to reflect the class being tested. E.g. test_geo_mechanics_newton_raphson_erosion_process_strategy.

Reply:
The name is changed,